src/types.h

```diff
@@ -470,15 +477,97 @@ static RenderTexture *jaylib_getrendertexture(const Janet *argv, int32_t n) {
     return ((RenderTexture *)janet_getabstract(argv, n, &AT_RenderTexture));
 }
 
+int camera2d_get(void *p, Janet key, Janet *out);
+void camera2d_put(void *p, Janet key, Janet value);
+
 static const JanetAbstractType AT_Camera2D = {
     "jaylib/camera-2d",
+    NULL,
+    NULL,
+    camera2d_get,
+    camera2d_put,
     JANET_ATEND_NAME
 };
 
 static Camera2D *jaylib_getcamera2d(const Janet *argv, int32_t n) {
     return ((Camera2D *)janet_getabstract(argv, n, &AT_Camera2D));
 }
 
+int camera2d_get(void *p, Janet key, Janet *out) {
+  Camera2D *camera = (Camera2D *)p;
+
+  if (!janet_checktype(key, JANET_KEYWORD)) {
+    janet_panic("expected keyword");
+  }
+
+  const uint8_t *kw = janet_unwrap_keyword(key);
+
+  if (!janet_cstrcmp(kw, "target")) {
```

Comment:
I am confused by the negation of the comparison here. I believe that the `!` is wrong here.

Reply:
I was confused by this as well, however I just copied whats being done in the camera2d constructor https://github.com/janet-lang/jaylib/blob/master/src/core.h#L749